m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2095,25 +2093,35 @@ PROCEDURE BUILDSLTRANSFERLIST
          TRANCREATEDATE(TRANCOUNT)=TRANSFERCREATEDATE
          TRANCREATEUSER(TRANCOUNT)=TRANSFERCREATEUSER
 
+         BNODEBUGMSG=FORMAT("**** TESTING FOR READONLY DATA - SHARE TRANSFER LOC ########9",TARGETTRANSFERLOC)
+         CALL BNODEBUGMSGADD
+         BNODEBUGMSG=FORMAT("EXTRECFOUND:9  ",EXTRECFOUND)+
+                     FORMAT("PARAMALLOWEDIT:9  ",PARAMALLOWEDIT)+
+                    "   SHARE TRAN FREQ:"+
+                     TRANSFERFREQCHR(TRANFREQALT,SHARE TRANSFER:FREQUENCY)+
+                    FORMAT("  TRANSFERCREATEUSER:,###9+  ",TRANSFERCREATEUSER)+
+                    FORMAT("USERSWHOCANEDIT 9999",TRANSFERCREATEUSER)+
```

Comment:
Hi @ Tom are you sure line 2103 of debug is correct? It says "FORMAT("USERSWHOCANEDIT 9999",TRANSFERCREATEUSER)". My guess is you meant to display a variable or array element having to do with USERSWHOCANEDIT instead of TRANSFERCREATEUSER. The two lines in a row indicates you already put TRANSFERCREATEUSER in the Debug part just before Line 2103:

                    FORMAT("  TRANSFERCREATEUSER:,###9+  ",TRANSFERCREATEUSER)+
                    FORMAT("USERSWHOCANEDIT 9999",TRANSFERCREATEUSER)+

Reply:
@jordan-keenan Corrected in next commit